python/sqlflow_playground/playground_server_design.md

```diff
@@ -0,0 +1,135 @@
+# SQLFlow Playground Server
+
+SQLFlow Playground Server exposes a REST API service that enables users to share
+the resources in one playground cluster. Users can take advantage of SQLFlow by
+installing a small [plugin](https://github.com/sql-machine-learning/pysqlflow)
+on her/his Jupyter Notebook.
+
+This service is used to extend the SQLFlow Playground's capability, especially
+when we need to manage the resource in the k8s cluster. We suppose the playground
+as a pure backend service (without Jupyter/JupyterHub) which provides machine 
+learning capability for some frontend. It clearly is not for those who just want
+to connect to the SQLFlow server in the playground through our built-in Jupyter 
+Notebook. Currently, this service is used to run our tutorials on [Aliyun
+DSW for Developer](https://dsw-dev.data.aliyun.com/) which behaves as a frontend
+of our playground.
+
+## The Architecture
+
+**SQLFlow Playground Server** is a side-car service of our playground cluster.
+Now, it is designed as an HTTP server which receives user login, creates DB
```

Comment:
In the following documentation, it appears that there is no `user login`, but `user authorization`?